config/config-observability.yaml

```diff
@@ -28,6 +28,7 @@ data:
 
   # The fluentd sidecar image used to collect logs from /var/log as a sidecar.
   # Must be presented if logging.enable-var-log-collection is true.
+  # KEEP THIS CONSISTENT WITH 999-cache.yaml
```

Comment:
TODO() add a test or have some way to add a cache entry for each per-task sidecar, which would cover Istio, too?

Reply:
Done